### PR TITLE
Initial commit for Spanish language for basic pages.

### PR DESCRIPTION
**Pull request by yeyeto2788**
Hello guys! I'm doing the tranlations to Spanish for the website. I'm doing it step by step.

I have left some links to the `en` language since I haven translated them. I'll be updating them as soon as I go deeper on it.

Regards.

P.D: I hope this is not comming out of the blue and if there is any issue just let me know.

### CODE REVIEW

File: es/discuss.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Discutir sobre Mu
+i18n: es
+---
+
+<h1>Discutir sobre Mu</h1>
```
Comment (carlosperate) [inline es/discuss.html]: This is probably more of a personal view, but to me `Discutir` sounds more like "arguing" than "talking", perhaps we can use a synonum like "Hablar"?

File: es/discuss.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<p>Es bueno hablar, compartir conocimientos y aprender de otros usuarios de Mu. Quizás
+¿Tiene alguna pregunta, problema o no estás seguro de cómo hacer algo con Mu?
+Discutir tal cosas en un ambiente de apoyo es una buena manera de hacer progreso,
+haz amigos y haz las cosas con éxito. En pocas palabras, Mu es un
+proyecto enfocado en la comunidad y hemos creado un grupo de discusión en línea
+donde la comunidad puede unirse.</p>
+
+<p>Las contribuciones son bienvenidas sin perjuicio de cualquier persona, independientemente de la edad,
+género, religión, raza o sexualidad. Si estás pensando, "pero no significan
```
Comment (carlosperate) [inline es/discuss.html]: Perhaps change `pero no significan yo` to `pero eso no me incluye`?

File: es/discuss.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<p>Nuestra plataforma de discusión es proporcionada por un servicio llamado "Gitter" y
+puedes registrarte y conectarte con la comunidad haciendo clic en el botón de abajo o
+visitando: <a href="https://gitter.im/mu-editor/general" target="_blank">https://gitter.im/mu-editor/general</a></p>
+<p><a href="https://gitter.im/mu-editor/general" target="_blank" class="btn btn-primary btn-lg">Conectarse al servicio de discusiones</a></p>
+
+<div class="alert alert-info" role="alert">
+<h4><strong>Código de conducta de la comunidad</strong></h4>
+<p>Esperamos que los colaboradores sigan el
+<a href="https://www.python.org/psf/codeofconduct/" target="_blank"><strong>Código de conducta</strong></a> de la fundación de Python Software (Python Software foundation).</p>
```
Comment (carlosperate) [inline es/discuss.html]:
```suggestion
<a href="https://www.python.org/psf/codeofconduct/" target="_blank"><strong>Código de conducta</strong></a> de la fundación de Python Software (Python Software Foundation).</p>
```

File: es/download.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+Hay muchas maneras de instalar Mu. Lo más sencillo es descargar la versión oficial
+del instalador para Windows o Mac OSX. Si encuentra que no puede instalar Mu porque el
+ordenador que está utilizando está bloqueado, debe probar PortaMu: un método que permite
+ejecutar Mu desde un pendrive en Windows u OSX. También puedes usar la
+herramienta `pip` incorporada de Python. Algunas distribuciones de Linux vienen con Mu empaquetado
+ya (y debe usar el administrador de paquetes de su sistema operativo para instalarlo). Finalmente,
+si está en Raspbian (la versión de Linux para Raspberry Pi) puede instalar Mu como un paquete.
+
+Si eres un desarrollador, puedes encontrar el código fuente.
+[on GitHub](https://github.com/mu-editor/mu).
```
Comment (carlosperate) [inline es/download.md]:
```suggestion
[en GitHub](https://github.com/mu-editor/mu).
```

File: es/download.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Hay muchas maneras de instalar Mu. Lo más sencillo es descargar la versión oficial
+del instalador para Windows o Mac OSX. Si encuentra que no puede instalar Mu porque el
+ordenador que está utilizando está bloqueado, debe probar PortaMu: un método que permite
+ejecutar Mu desde un pendrive en Windows u OSX. También puedes usar la
+herramienta `pip` incorporada de Python. Algunas distribuciones de Linux vienen con Mu empaquetado
+ya (y debe usar el administrador de paquetes de su sistema operativo para instalarlo). Finalmente,
+si está en Raspbian (la versión de Linux para Raspberry Pi) puede instalar Mu como un paquete.
+
+Si eres un desarrollador, puedes encontrar el código fuente.
```
Comment (carlosperate) [inline es/download.md]:
```suggestion
Si eres un desarrollador, puedes encontrar el código fuente
```

File: es/download.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+<br/>
+
+<div class="panel panel-danger">
+    <div class="panel-heading"><h3 class="panel-title">ATENCIÓN A LOS USUARIOS DE LINUX!</h3></div>
+    <div class="panel-body">
+    <p>En Linux, para que Mu funcione con los dispositivos basados en MicroPython
+     debe asegurarse de agregar al grupo de permisos correcto (por lo general,
+     los grupos <code> dialout </code> o <code> uucp </code>). También hay que
+     asegurarse de que su distribución monta automáticamente dispositivos flash, o
+     Asegúrate de montarlos manualmente.</p>
```
Comment (carlosperate) [inline es/download.md]:
```suggestion
     asegúrate de montarlos manualmente.</p>
```

File: es/about.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    <a class="btn btn-lg btn-info" href="/en/howto" role="button">Cómo ...?</a>
+</div>
+<img src="/img/logo.png" alt="Mu's world" class="img-responsive center-block img-rounded"/>
+
+
+<div class="row marketing">
+  <div class="col-lg-6">
+    <h4>Menos es Más.</h4>
+    <p>Mu tiene solo las características más esenciales, por lo que los usuarios no se sienten intimidados por una interfaz desconcertante.</p>
+    <h4>Construye el camino con la menos resistencia.</h4>
```
[the remark this answers is not in the document]
Comment (yeyeto2788) [reply es/about.html]: From my point of view `pisando` wouldn't fit on the phrase.

I would put something like: `Construye el camino con la menor resistencia`, take into account that there is also a typo (`camino con la menos resistencia`)

Note the `s` on the word `menos` it should also be changed to `menor`

What do you think?

File: es/discuss.html (hunk 2 above)
Comment (yeyeto2788) [reply es/discuss.html]: I actually do the changes from the computer adding this change.

**Comment by yeyeto2788** [comment]
THANK YOU! @carlosperate

I highly appreciated the fact that you took the time to review this. I wasn't 100% sure of some things. I'll update the remaining comments in a single commit tomorrow and I'll try to add more things. 

If you have any other suggestion, please, do not hesitate on pointing them out.

Regards.

**Comment by yeyeto2788** [comment]
Hey @carlosperate, I have added another page and I have added some of the proposed solutions. There are 2 comments remaining that we might need to fix. If you know

* `es/discuss.html` regarding the translation of `Discuss`
* `es/about.html` about which is the best option for `Tread the Path of Least Resistance.`

File: es/about.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (carlosperate) [reply es/about.html]: Yeah, I agree `pisando` doesn't quite work here.
I think I prefer `Camina el sendero...` in this case, it's probably closer to the original.

File: es/discuss.html (hunk 1 above)
Comment (carlosperate) [reply es/discuss.html]: Was the word `forum` used in the original text? If that's the case `foro` sounds like a good translation to me.

About the word `discutir` in itself, I can't say there is a clear winner. `Hablar` is a good options, perhaps `conversar`? Whichever you prefer.

Comment (yeyeto2788) [reply es/discuss.html]: The `forum` word is not on the original text but for me sounds a bit better than `discutir`, in the other hand tough `conversar` sounds much better than `Hablar`.

I will change it to `conversar` in the mean time and if we can come up with a better one I'll change it again :)

File: es/about.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (yeyeto2788) [reply es/about.html]: Ok, changing it right now then.

File: es/howto/bugs.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  <br/>
+</div>
+
+Si alguna vez has escrito alguna línea de código, sabrás que todo software tiene
+[errores](https://www.history.navy.mil/our-collections/photography/numerical-list-of-images/nhhc-series/nh-series/NH-96000/NH-96566-KN.html).
+Este es el caso con Mu.
+
+Si crees que ha encontrado un error, simplemente haga clic en el botón de abajo
+(necesita tener una cuenta en GitHub) y luego haga clic en el botón verde "Nuevo problema"
+Se encuentra en la parte superior derecha de la página:
```
[the remark this answers is not in the document]
Comment (yeyeto2788) [reply es/howto/bugs.md]: It has a typo, it should say:
`que se encuentra en la parte superior...`

**Comment by yeyeto2788** [comment]
@carlosperate, @ntoll  I think there is no pending reviews on this PR. I will be working again on translating next week. You can merge these changes if you guys want.

I'll add another PR for rest.

Regards.

**Comment by yeyeto2788** [comment]
@carlosperate, @ntoll by any chance, do you guys have images of mu in Spanish? I'm translating other pages but I guess that I'll need some pictures to be in Spanish.

Regards.

**Comment by carlosperate** [comment]
If the local language in the OS is set to Spanish it should automatically
launch in that language.
Alternatively if you are running from source code you can edit the __init__
file and set the language manually to Spanish.

**Comment by yeyeto2788** [comment]
@carlosperate great, I'll try to get those images out of running Mu locally on my environment. Hope it is ok.

If I get those images we'll need to define some things regarding directories and links. I hope there is no limit on hosting (Take into account that the more images the more space is needed)

Regards.

**Comment by carlosperate** [comment]
About the hosting no need to worry, the website is hosted by GitHub pages, so it is free. However, is always important to think about the website loading time for the users, so we should always try to keep the image sizes reasonably low (no need to make the images tiny, but always compress them to jpg/png or similar).

**Pull request closed without merging**